src/System.Windows.Forms/src/System/Windows/Forms/Controls/DataGridView/DataGridViewButtonCell.cs

```diff
@@ -17,7 +17,11 @@ public partial class DataGridViewButtonCell : DataGridViewCell
     private static readonly int s_propButtonCellFlatStyle = PropertyStore.CreateKey();
     private static readonly int s_propButtonCellState = PropertyStore.CreateKey();
     private static readonly int s_propButtonCellUseColumnTextForButtonValue = PropertyStore.CreateKey();
-    private static readonly VisualStyleElement s_buttonElement = VisualStyleElement.Button.PushButton.Normal;
+#pragma warning disable WFO5001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+    private static readonly VisualStyleElement s_darkButtonElement = VisualStyleElement.CreateElement("DarkMode_Explorer::BUTTON", 1, 1);
```

Comment:
I believe this was missed
```suggestion
    private static readonly VisualStyleElement s_darkButtonElement = VisualStyleElement.CreateElement($"{Control.DarkModeIdentifier}_{Control.ExplorerThemeIdentifier}::BUTTON", 1, 1);
```